lib/ash/expr/expr.ex

```diff
@@ -823,6 +823,11 @@ defmodule Ash.Expr do
     soft_escape(Ash.CiString.sigil_i(str, mods), escape?)
   end
 
+  def do_expr({:fragment, opts, [{_, _, [{:<<>>, _, [query]}, []]} | rest]}, escape?)
+      when is_binary(query) do
+    do_expr({:fragment, opts, [Macro.expand(query, __ENV__) | rest]}, escape?)
```

Comment:
So this discards whatever compile time transformations are made by the sigil doesn't it? I think we'd want to pass the AST through right?

Reply:
Good catch, I made a few changes that should address that properly.